lib/cisco_node_utils/snmpcommunity.rb

```diff
@@ -50,12 +66,22 @@ def destroy
     #  end
 
     def group
-      config_get('snmp_community', 'group', @name)
+      if platform == :nexus
+        config_get('snmp_community', 'group', name: @name)
```

Comment:
I am fairly certain this will break nxos.  You are using printf style wildcards in the yaml file but key-value wildcards when calling `config_get`.

https://github.com/cisco/cisco-network-node-utils/pull/315/files#diff-3e8591eaf966384e60ee4ec16e769fd8R38

Please update the yaml file to support key-value wildcards and test on both nxos/xr